Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
Good point. It doesn't print properly in the CircleCI logs either. I'll use colour codes instead:

```console
$ echo $'\e[33m\e[7mWARN\e[27m Doctests are only run in Node v6.x.x (current version is '"$(node --version)"$')\e[0m'
```